tests/accounts/adapter.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import pytest
+
+from allauth.account.adapter import get_adapter
+
+
+@pytest.mark.django_db
+def test_account_adapter_unicode():
+    adapter = get_adapter()
+    adapter.clean_username('ascii')
+    adapter.clean_username(u'unicöde')
```

Comment:
Can we add a user that's actually beyond latin1?

I'm worried that this won't test the user in other contexts, e.g. user stats, submissions, suggestions, etc.